index.html

```diff
@@ -38,7 +39,10 @@ <h2 id="booklisttitle">ALL AWESOME BOOKS</h2>
     </div>
   </div>
   </section>
-  
+  <div class="current-date">
```

Comment:
- [ ] I have realized that the date is only visible when you are on the "List all books" page and not visible in the other "Add book" and "contact" pages. Kindly ensure that it is visible in all three pages.

![image](https://user-images.githubusercontent.com/86318284/200791312-ece15431-fdcc-4110-800b-f64778fce1b9.png)

![image](https://user-images.githubusercontent.com/86318284/200791380-787dbd23-68de-4275-929f-05826a68f8d4.png)